config/sunspot.yml

```diff
@@ -6,7 +6,7 @@ production:
     read_timeout: 120
     log_level: WARNING
     path: "/solr/default"
-  disabled: true
+  disabled: false
```

Comment:
So to merge this, weould we want to undo this line and leave it `disabled: true` until we resolve the reindexing issue?